packages/flutter/test/material/search_test.dart

```diff
@@ -496,6 +496,17 @@ void main() {
                 label: routeName,
                 textDirection: TextDirection.ltr,
                 children: <TestSemantics>[
+                  TestSemantics(
```

Comment:
If this isn't correct, we have to put a sort key on it I think.

Reply:
This seems wrong if things are dumped in traversal order (which I think they are).

I am surprised that it would need a sort key, though. The suggestions come physically after the back button and the text field.

(We should verify that talkback and voiceover do the right thing here on device)

Reply:
This is actually how things are ordered today.  It's not very clear to me why that's the case - even without any changes here.  Filed https://github.com/flutter/flutter/issues/34432 and added a TODO.

Reply:
Talked offline - will close the other bug.  Traversal order test was broken and needed to be updated.